http-clients/apache5-client/src/main/resources/META-INF/services/software.amazon.awssdk.http.SdkHttpService

```diff
@@ -0,0 +1,46 @@
+#
```

Comment:
Multiple copies of the license header here

Reply:
Removed it now , this got copied over while migrating from https://github.com/aws/aws-sdk-java-v2/blob/master/http-clients/apache-client/src/main/resources/META-INF/services/software.amazon.awssdk.http.SdkHttpService